cli/flox/src/utils/message.rs

```diff
@@ -9,6 +9,30 @@ fn print_message(v: impl Display) {
     eprintln!("{v}");
 }
 
+#[allow(dead_code)]
+#[derive(Debug, PartialEq)]
+pub enum Action {
+    Plain(String),
+    Error(String),
+    Created(String),
+    Deleted(String),
+    Updated(String),
+    Warning(String),
+}
+
+impl Action {
+    pub fn print(self) {
```

Comment:
print can act on a reference alone, no need to move (and drop) the object.

(`v` will be &String, which implements `Display`)

```suggestion
    pub fn print(&self) {
```